snippets/acknowledge-the-update.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: SNIPPET
+[IMPORTANT]
+====
+Before you acknowledge the update, verify that there are no Kubernetes APIs in use that are removed in the version you are updating to.
+For example, in {product-title} 4.17, there are no API removals.
+See "Kubernetes API removals" for more information.
+====
+
+.Prerequisites
+
+* You have verified that APIs for all of the applications running on your cluster are compatible with the next Y-stream release of {product-title}.
+For more details about compatibility, see "Verifying cluster API versions between update versions".
+
+.Procedure
+
+. Perform the administrative acknowledgment to start the cluster update.
```

Comment:
```suggestion
. Complete the administrative acknowledgment to start the cluster update by entering the following command:
```
IBM Style advises to avoid "perform": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#perform
OCP docs advise to introduce a command with explicit guidance to "run" or "enter" the command: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command